src/rdpc/types.ts

```diff
@@ -0,0 +1,35 @@
+import { RdpcDonorInfo } from "indexClinicalData/types";
+
+export interface Analysis {
+  analysisId: string;
+  analysisType: string;
+  donors: Donor[];
+  runs: Run[];
+}
+
+export interface SimpleAnalysis {
+  analysisId: string;
+}
+
+export interface Donor {
+  donorId: string;
+}
+
+export interface Run {
+  runId: string;
+  state: string;
+  repository: string;
+  inputAnalyses: SimpleAnalysis[];
+}
+
+export interface InputAnalysesRunMap {
+  [inputAnalyses: string]: Run[];
+}
```

Comment:
I think I'd name this `RunsByInputAnalyses`, since the content of it is runs grouped by inputAnalyses